guide/01-getting-started/install-and-set-up.ipynb

```diff
@@ -229,7 +229,7 @@
     "\n",
```

Comment:
<p>Both&nbsp;<code>conda install -c esri arcgis</code>&nbsp;and&nbsp;<code>pip install arcgis</code>&nbsp;will install all of the dependencies outlined in the&nbsp;<a href="https://developers.arcgis.com/python/guide/system-requirements/" target="_blank">system requirements</a>&nbsp;section. However, the API can function in a 'stripped down' state with only a few dependencies. This allows developers to deploy the API with a minimal footprint in space constrained environments. For this to work, you need to manually manage the dependencies in your environment based on the problems you need to solve and on which features of the API you want to utilize. As of&nbsp;<code>v1.8.3</code>&nbsp;of the Python API, the minimum dependencies needed are:</p><ul><li><code>ujson</code></li><li><code>six</code></li><li><code>requests</code></li><li class="ql-indent-1">This library has its own dependencies that must be included</li><li><code>requests_toolbelt</code></li><li><code>requests_ntlm</code></li><li class="ql-indent-1">This library has its own dependencies that must be included</li><li><code>ntlm_auth</code></li></ul><p>To install the API with no dependencies, simply add the&nbsp;<code>--no-deps</code>&nbsp;flag to any install command, i.e.&nbsp;<code>conda install -c esri arcgis --no-deps</code>&nbsp;or&nbsp;<code>pip install arcgis --no-deps</code>.</p><p>The above set of dependencies should allow you to establish a connection with your web GIS and perform a number of administrative and other tasks that involve processing on your GIS, rather than in your Python kernel. Depending on your OS, this set up with minimal dependencies should take up about&nbsp;<code>50</code>&nbsp;mb in space. As your needs expand, you can manually choose which dependencies to add to your Python environment. For instance, you can install&nbsp;<code>pandas</code>, if you need to work with spatially enabled DataFrames, and&nbsp;<code>jupyter</code>, if you need the notebook IDE.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Esri/arcgis-python-api/pull/1124/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='22'/>